src/ashish_Vyas/Assignment_15/BankusingSwitch.java

```diff
@@ -0,0 +1,173 @@
+package ashish_Vyas.Assignment_15;
+
+import java.util.Scanner;
+
+class BankusingSwitch{
+		double iciciBal = 25000;
+		double hdfcBal = 38000;
+		double sbiBal = 59000;
+		double bobBal = 48000;
+		
+		int icicidebitcount, hdfcdebitcount, sbidebitcount, bobcreditcount ;
+		int icicicreditcount, hdfccreditcount, sbicreditcount, bobdebitcount;
+		
+		static int totaldebitcount;
+		static int totalcreditcount;
+
+void operation(String bankname, String operation, double amount){
+	
+	switch(operation.toUpperCase()){
+	
+		case "DEBIT" :
+			switch(bankname.toUpperCase()){
+				case "ICICI" :
+					if(iciciBal>=amount){
```

Comment:
nstead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)